sky/packages/sky/lib/src/widgets/snack_bar.dart

```diff
@@ -38,7 +38,7 @@ class SnackBarAction extends Component {
   }
 }
 
-class SnackBar extends Component {
+class SnackBar extends StatefulComponent {
```

Comment:
Actually AnimatedComponent is a bit weird. It's only used by AnimatedContainer (why? it doesn't seem to actually use the AnimatedComponent logic; see also #1100), MimicOverlay, which actually manages a Performance itself, and so could be reframed in terms of a superclass that manages one, and transitions.

Maybe it should be merged into Transitions, and then we could use the term AnimatedComponent to mean a Component that has its own AnimationPerformance, which would mean SnackBar, MimicOverlay, and maybe even Drawer and PopupMenu could use it.